aut/commands/validator.py

```diff
@@ -49,7 +49,14 @@ def info(rpc_endpoint: Optional[str], validator_addr_str: str) -> None:
 
     validator_addr = get_validator_address(validator_addr_str)
     aut = autonity_from_endpoint_arg(rpc_endpoint)
-    print(to_json(aut.get_validator(validator_addr), pretty=True))
+    validator_data = aut.get_validator(validator_addr)
+    if validator_data is None or validator_data.get("addr", "") != validator_addr_str:
+        echo(
+            f"The address {validator_addr_str} is not registered as a validator.",
+            err=True,
+        )
+        return
+    echo(to_json(aut.get_validator(validator_addr), pretty=True))
```

Comment:
As a rule, all errors should cause a command to exit with non-zero status. But in the case of a non-existent validator, `aut` still exits with `0` (success).